cmis_stock_production_lot/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
you should use the new way to write the README files https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst